src/components/Search/SearchRouter/SearchRouterInput.tsx

```diff
@@ -1,10 +1,11 @@
-import React, {useState} from 'react';
 import type {ReactNode, RefObject} from 'react';
+import React, {useState} from 'react';
+import type {StyleProp, TextInput as TextInputRef, ViewStyle} from 'react-native';
```

Comment:
NAB
```suggestion
import type {StyleProp, TextInput, ViewStyle} from 'react-native';
```

Reply:
Maybe, instead of `TextInputRef`, `RNTextInput` would be a better name here.

Reply:
Agree that using `RNTextInput` would be better in that case